content/articles/how-to-create-a-user-login-page-using-javagui/index.md

```diff
@@ -0,0 +1,136 @@
+### How to create a user login page using Java GUI
+
+### Introduction
+
+Java is an Object-Oriented Programming Language. G.U.I stands for Graphical User Interface. GUIs are made up of objects and classes that can interact with each other. GUIs are said to be `Event Driven` programs. this means that GUIs depends on series of object events that can occur at any given time.
```

Comment:
A more detailed intro could be given.

Speak more about Java. How it started as a backend framework? It's evolution into a frontend framework. What GUIs are? Where do they use it? What other languages can you build GUIs?

Give them the history behind building GUIs with Java.